brave-spancollector-kafka/src/main/java/com/github/kristofa/brave/kafka/KafkaSpanCollector.java

```diff
@@ -0,0 +1,100 @@
+package com.github.kristofa.brave.kafka;
+
+import com.github.kristofa.brave.SpanCollector;
+
+import java.io.Closeable;
+import java.util.Properties;
+import java.util.concurrent.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.twitter.zipkin.gen.Span;
+import kafka.javaapi.producer.Producer;
+import kafka.producer.ProducerConfig;
+
+/**
+ * SpanCollector which submits spans to Kafka using <a href="http://kafka.apache.org/documentation.html#producerapi">Kafka Producer api</a>.
+ * <p>
```

Comment:
add: <p/>Spans are sent to kafka as keyed messages: the key is the topic zipkin and the value is a TBinaryProtocol encoded Span.